alpa/device_mesh.py

```diff
@@ -97,9 +101,86 @@ def __init__(self, server_address: str, num_hosts: int, host_id: int):
                         jnp.ones((1,), dtype=jnp.int8), d),
                                        take_ownership=True))
 
+    ##### TensorStore Related Functions #####
+    def get_tensorstore_spec(self, ckpt_path: str):
+        spec = {'driver': 'zarr', 'kvstore': {}, 'metadata_key': f".zarray{self.host_id}"}
+        if ckpt_path.startswith('gs://'):
+            m = re.fullmatch('^gs://([^/]*)/(.*)$', ckpt_path, re.DOTALL)
+            if m is None:
+                raise ValueError('The ckpt_path should contain the bucket name and the '
+                                f'file path inside the bucket. Got: {ckpt_path}')
+            gcs_bucket = m.group(1)
+            path_without_bucket = m.group(2)
+            spec['kvstore'] = {'driver': 'gcs', 'bucket': gcs_bucket,
+                            'path': path_without_bucket}
+        else:
+            spec['kvstore'] = {'driver': 'file', 'path': ckpt_path}
+        return spec
+
     ##### Buffer Related Functions #####
+    def view_buffer(self, uuid: int):
```

Comment:
This function is redundant.
Use `get_buffer` instead

Reply:
This function is debug-only, I will remove it after I implement model save/load. Leave this as resolved to remind me.